h_transport_materials/property_database/steel_316L/steel_316L.py

```diff
@@ -246,11 +255,84 @@
 
 serra_solubility = Solubility(
     data_T=1 / serra_data["solx"] * htm.ureg.K,
-    data_y=serra_data["soly"] * htm.ureg.mol * htm.ureg.m**-3 * htm.ureg.Pa**-0.5,
+    data_y=serra_data["soly"] * SOLUBILITY_UNIT_MOL_VOLUME,
     isotope="H",
     source="serra_hydrogen_2004",
 )
 
+pomellalobo_data = {
+    "source": "pomellalobo_2022",
+    "isotope": "D",
+    "T": [100, 350, 400],
+    "T_unit": "ºC",
+    "D_0": [7.17e-7, 8.18e-7, 5.14e-7],
+    "D_0_unit": "m2.s-1",
+    "E_D": [42.50, 42.50, 42.50],
+    "E_D_unit": "kJ.mol-1",
+    "S_0": [1.50, 5.50e-1, 4.20e-1],
+    "S_0_unit": "mol.m-3.Pa-0.5",
+    "E_S": [12.87, 17.50, 15.95],
+    "E_S_unit": "kJ.mol-1",
+    "mean_note": (
+        "Arithmetic mean for activation energies and geometric mean for "
+        "pre-exponential coefficients; mean of experimental results not "
+        "present in original reference."
+    ),
+}
+
+pomellalobo_properties = []
+for t, temperature in enumerate(pomellalobo_data["T"]):
+    pomellalobo_properties.append(
+        Diffusivity(
+            D_0=pomellalobo_data["D_0"][t] * htm.ureg.m**2 * htm.ureg.s**-1,
+            E_D=pomellalobo_data["E_D"][t] * htm.ureg.kJ * htm.ureg.mol**-1,
+            isotope=pomellalobo_data["isotope"],
+            source=pomellalobo_data["source"],
+            note=(
+                "Coefficients derived from experimental data at "
+                f"temperature {temperature} {pomellalobo_data['T_unit']}"
+            ),
+        )
+    )
+    pomellalobo_properties.append(
+        Solubility(
+            S_0=pomellalobo_data["S_0"][t] * SOLUBILITY_UNIT_MOL_VOLUME,
+            E_S=pomellalobo_data["E_S"][t] * htm.ureg.kJ * htm.ureg.mol**-1,
+            isotope=pomellalobo_data["isotope"],
+            source=pomellalobo_data["source"],
+            note=(
+                "Coefficients derived from experimental data at "
+                f"temperature {temperature} {pomellalobo_data['T_unit']}"
+            ),
+        )
+    )
```

Comment:
I don't understand these properties, I must admit I haven't read the paper in full though.

These properties are obtained at one temperature, correct?

Should we just add 2 properties (a diffusivity and solubility) based on the three experimental points from each? We could directly fit the three points by an Arhenius law.

Something like:

```python
  Diffusivity(
      data_T=htm.ureg.Quantity([100, 350, 400], htm.ureg.degC),
      data_y=[..., ..., ...] * htm.ureg.m**2 * htm.ureg.s**-1,
      isotope="D",
      source="pomellalobo_2022",
      note=pomellalobo_data["mean_note"],
  )
```